packages/backend-lib/src/journeys.ts

```diff
@@ -257,13 +270,83 @@ export async function getJourneysStats({
     journeysStats.push(stats);
 
     for (const node of definition.nodes) {
+      if (
+        node.type === JourneyNodeType.RateLimitNode ||
+        node.type === JourneyNodeType.ExperimentSplitNode
+      ) {
+        continue;
+      }
+
+      const nodeStats = statsMap.get(node.id) ?? new MapWithDefault(0);
+
+      if (node.type === JourneyNodeType.SegmentSplitNode) {
+        const parentNodeProcessed = nodeProcessedMap.get(node.id);
+        const falseChildNodesProcessed =
+          nodeProcessedMap.get(node.variant.falseChild) ?? 0;
+
+        if (parentNodeProcessed) {
+          const falseChildNodeProcessedRate =
+            falseChildNodesProcessed / parentNodeProcessed;
+          const falseChildEdgeProportion =
+            (falseChildNodeProcessedRate > 1
```

Comment:
Can you explain when this would ever be greater than 1?

Reply:
@maxgurewitz I think I have added this check due to some inconsistent manual entries in my DB earlier. But when I automated the event triggering after running worker, then this check is not required. Hence removed it.